conftest.py

```diff
@@ -155,7 +155,7 @@ def repository():
 
     with runner.isolated_filesystem() as project_path:
         result = runner.invoke(
-            cli, ['init', '.', '--template', 'Basic Python Project'],
+            cli, ['init', '.', '--template-id', 'python-minimal'],
```

Comment:
I know that this is decided somewhere else, but is there a reason to have the `-minimal` postfix here? It would be cleaner to have just `python` and `R`. Also, `--template` looks cleaner to me than `--template-id`.

Reply:
for `-minimal` I tend to agree with you but that should be changed here first https://github.com/SwissDataScienceCenter/renku-project-template
We may open an issue there.

We split `--template` in `--template-id` and `--template-index` in this PR, I think `--template` was confusing and we agreed on this type of change during the last design meeting. We can further change this if it turns out not to be clear enough, but I would rather avoid using `--template` since there are many options named `--template-xxx`